README.md

```diff
@@ -43,21 +43,21 @@ npm install
 
 ## New Token Methods
 
-Líf token is ERC20 compatible but it also has two more methods to allow the transference of data between users/contracts.
+Líf token is ERC20 compatible but it also has two more methods to allow the transference of data and execution of calls between users/contracts.
 
 ### transferData
 
 Transfer tokens from one address to another.
 ```
-transferData(address _to, uint _value, string _data) => (bool success)
+transferData(address to, uint value, bytes data, bool doCall) => (bool success)
```

Comment:
> we are near the gas limit on contract deployment

Interesting. How does this work? I mean I read a bit about the gas cost and limits, I have a rough idea of what you wanted to mean, but how do you near that we are near the limit? How do you measure it?